{
  "contest": 162,
  "handle": "KIntern_NA",
  "address": "0x9cD641c50320Ad533a07f6bAFE8b313599D071AE",
  "risk": "2",
  "title": "Users's funds can be drained because of wrong condition in function `withdraw()`",
  "issueId": 281,
  "issueUrl": "https://github.com/code-423n4/2022-09-y2k-finance-findings/issues/281"
}